packages/Autocomplete/src/index.tsx

```diff
@@ -0,0 +1,84 @@
+import Input from "@blaze-react/input";
+import utils from '@blaze-react/utils';
+import React, { Fragment, useState } from "react";
+
+interface IAutocompleteProps {
+  data: {
+    keyValue: string;
+    filterBy: any[];
+    data: object[];
+  };
+  utils: {
+    uniqueId: (element: any) => string
+  };
+  selected: (...args: any[]) => any;
+  placeholder?: string;
+  children?: any;
+}
+const Autocomplete: React.SFC<IAutocompleteProps> = ({
+  data: { data, filterBy: keys, keyValue },
+  placeholder,
+  selected,
+  utils: {
+    uniqueId
+  },
+  children
+}) => {
+  const [inputValue, setInputValue] = useState('');
```

Comment:
You could type useState also I think doing useState<string>(''). You will find an example here: https://medium.com/@jrwebdev/react-hooks-in-typescript-88fce7001d0d